fairscale/optim/oss.py

```diff
@@ -654,11 +649,3 @@ def _setup_bucket_strategy(self) -> None:
 
                 # Resize the bucket to remove lost space in the end
                 self.buckets[device][dst_rank].resize_(offset)
```

Comment:
this was good as such, but it meant that this method had side effects (bucket resize), so it would break if new parameters which could have fit in the original bucket were added